web/src/components/HeaderActionItemDropDown.tsx

```diff
@@ -35,6 +34,7 @@ export const Container = styled.div`
     object-fit: contain;
     width: 100%;
     height: 100%;
+    opacity: 0.85;
```

Comment:
can you give me a hint why this is needed?

Reply:
just to make the language change icon look more grey and less black I guess, should be fixed in https://issues.tuerantuer.org/browse/IGAPP-1287